docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -156,3 +139,13 @@ And you're done!
 You may choose to abstract code for `test.before`, `test.beforeEach`, `test.afterEach.always` and `test.after.always` into a separate file.
 
 To see a demo of this configuration file, look at https://github.com/zellwk/ava-mdb-test
+
+## Why `test.serial` instead of `test`
```

Comment:
When you use mongoose, you interact with the database through Mongoose. Since there can only be one Mongoose instance (and that Mongoose instance must always point to the same database), we cant access multiple databases even if we create them. 

Do I make sense?